Settings/modsettings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "Version": "1.5",
+    "Version": "2.0",

```

Comment:
I don't think you need to update the version if older settings are still compatible. You're causing users to reset their existing settings, when old settings are compatible with your changes.

Reply:
On the other hand, if I were to download version "2.0" of a mod from nexus and install it, only to see as version "1.5" inside of DFU, I'd be pretty confused.

Reply:
That's the version of settings, not the mod. @KABoissonneault is most likely right. So, I changed it back